handler/handler.go

```diff
@@ -50,7 +64,26 @@ func (q *NatsQueue) Queue(req *queue.Request) error {
 		log.Println(err)
 	}
 
-	err = q.nc.Publish(q.Topic, out)
+	err = q.stanConn.Publish(q.Topic, out)
```

Comment:
Note that the reconnect handler is async, so if you replace the connection there, and access it here, you need some locking to avoid races.